src/Types.jl

```diff
@@ -796,7 +796,7 @@ end
 
 function resolve_projectfile!(env::EnvCache, pkg, project_path)
     project_file = projectfile_path(project_path; strict=true)
-    project_file === nothing && pkgerror(string("could not find project file in package at `",
+    project_file === nothing && pkgerror(string("could not find project.toml file in package at `",
```

Comment:
I think just "project file" is used quite a lot, but perhaps:
```suggestion
    project_file === nothing && pkgerror(string("could not find project file (Project.toml or JuliaProject.toml) in package at `",
```